joplin/base/models/janis_page.py

```diff
@@ -257,6 +260,9 @@ def get_edit_handler(cls):
     class Meta:
         abstract = True
 
+# TODO: tech debt here, added an additional rule to hide this for location page
```

Comment:
Where did you add the additional rule?

also my branch (that I just merged to master) has some changes here, maybe that can be extended to hide for location